lib/dip/config.rb

```diff
@@ -112,6 +113,24 @@ def to_h
       end
     end
 
+    def validate
```

Comment:
I think we need an environment variable to skip the validation step. Something like `DIP_CONFIG_VALIDATION=false`

Reply:
That's a good idea. I'll add it.

Reply:
It's added